github/orgs_audit_log.go

```diff
@@ -0,0 +1,105 @@
+// Copyright 2021 The go-github AUTHORS. All rights reserved.
+//
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package github
+
+import (
+	"context"
+	"fmt"
+)
+
+// GetAuditLogOptions sets up optional parameters to query audit-log endpoint.
+type GetAuditLogOptions struct {
+	Phrase  *string `json:"phrase,omitempty"`  // A search phrase. (Optional.)
+	Include *string `json:"include,omitempty"` // Event type includes. Can be one of "web", "git", "all". Default: "web". (Optional.)
+	Order   *string `json:"order,omitempty"`   // The order of audit log events. Can be one of "asc" or "desc". Default: "desc". (Optional.)
+	ListCursorOptions
+}
+
+// AuditEntry describes the fields that may be represented by various audit-log "action" entries.
+// For a list of actions see - https://docs.github.com/en/github/setting-up-and-managing-organizations-and-teams/reviewing-the-audit-log-for-your-organization#audit-log-actions
+type AuditEntry struct {
+	Action               *string    `json:"action,omitempty"`
+	Active               *string    `json:"active,omitempty"`
+	ActiveWas            *string    `json:"active_was,omitempty"`
+	Actor                *string    `json:"actor,omitempty"`
+	BlockedUser          *string    `json:"blocked_user,omitempty"`
+	Business             *string    `json:"business,omitempty"`
+	CancelledAt          *Timestamp `json:"cancelled_at,omitempty"`
+	CompletedAt          *Timestamp `json:"completed_at,omitempty"`
+	Conclusion           *string    `json:"conclusion,omitempty"`
+	Config               *string    `json:"config,omitempty"`
+	ConfigWas            *string    `json:"config_was,omitempty"`
+	ContentType          *string    `json:"content_type,omitempty"`
+	CreatedAt            *int64     `json:"created_at,omitempty"`
+	DeployKeyFingerprint *string    `json:"deploy_key_fingerprint,omitempty"`
+	DocumentID           *string    `json:"_document_id,omitempty"`
+	Emoji                *string    `json:"emoji,omitempty"`
+	EnvironmentName      *string    `json:"environment_name,omitempty"`
+	Event                *string    `json:"event,omitempty"`
+	Events               *string    `json:"events,omitempty"`
```

Comment:
Just FYI - I'm going to just trust you on the types since this is not officially documented yet.
As usual, we can always update these when they change.